pkg/releasesync/releasesync.go

```diff
@@ -0,0 +1,232 @@
+package releasesync
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/openshift/sippy/pkg/db"
+	"github.com/openshift/sippy/pkg/db/models"
+	"k8s.io/klog"
+)
+
+type releaseSyncOptions struct {
+	db            *db.DB
+	httpClient    *http.Client
+	releases      []string
+	architectures []string
+}
+
+func Import(dbc *db.DB, releases, architectures []string) error {
+	o := releaseSyncOptions{
+		db:            dbc,
+		releases:      releases,
+		architectures: architectures,
+		httpClient:    &http.Client{Timeout: 60 * time.Second},
+	}
+
+	return o.Run()
+}
+
+func (r *releaseSyncOptions) Run() error {
+	for _, release := range r.releases {
+		fmt.Fprintf(os.Stderr, "Fetching release %s from release controller...\n", release)
```

Comment:
Suggest klog for consistency. 

I'd like to replace them all with logrus someday.